obspy/io/sac/sactrace.py

```diff
@@ -1056,6 +1056,9 @@ def read(cls, source, headonly=False, ascii=False, byteorder=None,
             beginning with '-12345' are considered unset. If True, they
             are instead passed without modification.  Good for debugging.
         :type debug_strings: bool
+        :param encoding: Encoding string that passes the user specified
+        encoding scheme.
+        :type encoding: str
```

Comment:
@jkmacc-LANL Thanks for catching it. Yeah, my IDE keeps adding white space in comments. I need to fix this problem every time I changed docstring.